public/javascript/projects.js

```diff
@@ -10,6 +10,10 @@
 
     buttonEls$.click(function() {
       var projectName = this.dataset.projectName;
+      
+      var confirm = window.confirm('Do you really want to delete project ' + projectName + '? This cannot be undone!')
```

Comment:
Looks good. Maybe add a note to make it clear that it is trivial to check projects out again? I'll leave that up to you for a future commit.